Http/HttpRequest.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using UnityEngine.Networking;
+
+namespace DUCK.Http
+{
+	public class HttpRequest
+	{
+		public UnityWebRequest UnityWebRequest { get; private set; }
+
+		private readonly Dictionary<string, string> headers;
+
+		public HttpRequest(UnityWebRequest unityWebRequest)
+		{
+			UnityWebRequest = unityWebRequest;
```

Comment:
This is very confusing, why is the member variable uppercased? It is the same as the class `UnityWebRequest`

Reply:
That's because its'a c# property and that is the correct naming convention for that. However, I agree that it looks a bit odd, but it's rather common nevertheless.

Reply:
Yeah properties should have upper cased names. Kevin says it's common but i'd go further pretty much all over C# land.

Reply:
[MS approved](https://msdn.microsoft.com/en-us/library/fzcth91k(v=vs.71).aspx)